examples/CORE/config.yaml

```diff
@@ -0,0 +1,210 @@
+programs:
```

Comment:
This file looks like a combination of multiple files, should it be here?

Reply:
It is better to have a config file that can be used by user directly, so I create this file.

Reply:
All examples are intended to be isolated single programs per config. The end users can combine multiple if they choose so, but that's something that happens outside of this repo.